omod/src/main/webapp/resources/htmlforms/HMIS-TB-003-TBClientCard-EncounterPage.xml

```diff
@@ -0,0 +1,418 @@
+<htmlform formName="HMIS TB 003: TB Client Card Encounter Page" formDescription="Admission into a clinic for a pregnant woman to give birth" formEncounterType="a9f11592-22e7-45fc-904d-dfe24cb1fc67" formUuid="0248cd00-bc53-11ea-b3de-0242ac130004" formVersion="3.0.0" formAddMetadata="yes" formUILocation="patientDashboard.visitActions" formOrder="9" formIcon="icon-medkit" formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; patient.person.gender=='F' &amp;&amp;sessionLocation.uuid=='e9bc61b5-69ff-414b-9cf0-0c22d6dfca2b'" formDisplayStyle="Standard" formLabel="HMIS-TB-003-TB CLIENT CARD - ENCOUNTER PAGE">
```

Comment:
1. Use encounter type 455bad1f-5e97-4ee9-9558-ff1df8808732 (TB Encounter (Followup))
1. Why is your location filter for MCH instead of TB location instead of the one for MCH

Reply:
Also indent the <htmlform tag attribute each on a separate line for readability

Reply:
Done